
# Details of input
string input_id

string status

int32 BUTTON_INACTIVE=0
int32 BUTTON_ACTIVE=1
int32 BUTTON_HOVER=1

int32 button_status 

float64 percent


